core/src/test/java/com/datastax/oss/driver/api/core/cql/StatementBuilderTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.core.cql;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class StatementBuilderTest {
+
+  private static class NullStatementBuilder
+      extends StatementBuilder<NullStatementBuilder, SimpleStatement> {
```

Comment:
I don't mind creating a class, but why not just use `SimpleStatementBuilder`? The only thing it needs is a query string.

Reply:
The functionality in question is defined in the abstract superclass so I wanted a way to get at it as cleanly as possible.  While admittedly quite unlikely there's always the possibility that a subclass could modify the behaviour... so I stayed away from SimpleStatementBuilder.